lib/carto_metadata.js

```diff
@@ -524,20 +524,20 @@ module.exports = function(redis_opts) {
         this.getMultipleUserLimitsParams(username, limitsParams, callback);
     };
 
-    me.rateLimitLuaScript = 
+    me.rateLimitLuaScript =
         'local results = {} ' +
         'local resultsCounter = 0 ' +
 
         'local limits = {} ' +
         'local limitsArray = redis.call("LRANGE", KEYS[1], 0, -1) ' +
-        
+
         'for i, v in ipairs(limitsArray) do ' +
         '    local rest = i % 3 ' +
         '    if rest ~= 0 then ' +
         '        limits[rest] = v ' +
         '    else ' +
         '        resultsCounter = resultsCounter + 1 ' +
-        '        results[resultsCounter] = redis.call("CL.THROTTLE", KEYS[2], limits[1], limits[2], v) ' +
+        '        results[resultsCounter] = redis.call("CL.THROTTLE", KEYS[2] .. resultsCounter, limits[1], limits[2], v) ' +
```

Comment:
What about `limits:rate:status:dgaubert:maps:tile:{{index}}` instead of `limits:rate:status:dgaubert:maps:tile{{index}}`??

Notice the use of `:` to separate `index` from the rest of the key.